airflow-core/src/airflow/ui/src/i18n/locales/ar/dags.json

```
@@ -0,0 +1,147 @@
+{
+  "assetSchedule": "تم تحديث {{count}} من أصل {{total}}",
+  "dagActions": {
+    "delete": {
+      "button": "حذف الDag",
+      "warning": "سيؤدي هذا إلى إزالة جميع البيانات الوصفية المرتبطة بالDag، بما في ذلك التشغيلات والمهام."
+    },
+    "trigger": {
+      "button": "تشغيل",
+      "triggerDag": "تشغيل الDag"
+    }
+  },
+  "filters": {
+    "paused": {
+      "active": "مفعّل",
+      "all": "الكل",
+      "paused": "متوقف"
+    }
+  },
+  "list": {
+    "advancedSearch": "بحث متقدم",
+    "clearSearch": "مسح البحث",
+    "columns": {
+      "dagId": "معرّف الDag",
+      "lastDagRun": "آخر تشغيل لـDag",
+      "nextDagRun": "التشغيل التالي لـDag",
+      "schedule": "الجدولة",
+      "tags": "العلامات"
+    },
+    "ownerLink": "رابط المالك لـ{{owner}}",
+    "searchPlaceholder": "البحث في الDags"
+  },
+  "runAndTaskActions": {
+    "clear": {
+      "button": "مسح {{type}}",
+      "buttonTooltip": "اضغط shift+c للمسح",
+      "dialog": {
+        "affectedTasks": {
+          "columns": {
+            "mapIndex": "فهرس الخريطة",
+            "runId": "معرف التشغيل",
+            "state": "الحالة",
+            "taskId": "معرف المهمة"
+          },
+          "noItemsFound": "لم يتم العثور على مهام.",
+          "title": "المهام المتأثرة: {{count}}"
+        },
+        "confirm": "تأكيد",
+        "note": {
+          "placeholder": "أضف ملاحظة...",
+          "title": "ملاحظة"
+        },
+        "options": {
+          "downstream": "المهام التابعة",
+          "existingTasks": "مسح المهام الحالية",
+          "future": "المستقبل",
+          "onlyFailed": "مسح المهام الفاشلة فقط",
+          "past": "الماضي",
+          "queueNew": "إضافة مهام جديدة للانتظار",
+          "upstream": "المهام السابقة"
+        },
+        "title": "مسح {{type}}"
+      }
+    },
+    "delete": {
+      "button": "حذف {{type}}",
+      "dialog": {
+        "resourceName": "{{type}} {{id}}",
+        "title": "حذف {{type}}",
+        "warning": "سيؤدي هذا إلى إزالة جميع البيانات الوصفية المرتبطة بـ{{type}}."
+      },
+      "error": "حدث خطأ اثناء الحذف {{type}}",
+      "success": {
+        "description": "تم تنفيذ طلب الحذف {{type}} بنجاح.",
+        "title": "{{type}} حُذف بنجاح"
+      }
+    },
+    "markAs": {
+      "button": "تمييز {{type}} كـ...",
+      "buttonTooltip": {
+        "failed": "اضغط shift+f لتمييز كفاشل",
+        "success": "اضغط shift+s لتمييز كناجح"
+      },
+      "dialog": {
+        "confirm": "تأكيد",
+        "options": {
+          "downstream": "المهام التابعة",
```

```suggestion
          "downstream": "المهام اللاحقة",
```
you can use `اللاحقة` as the opposite of `السابقة`